react-components/src/architecture/concrete/pointsOfInterest/color.ts

```diff
@@ -7,6 +7,7 @@ import { PointsOfInterestStatus } from './types';
 
 export const DEFAULT_OVERLAY_COLOR = new Color('#C945DB');
 export const PENDING_OVERLAY_COLOR = new Color('#33AA33');
+export const SELECTED_COLOR = new Color(0.392, 0.392, 1.0);
```

Comment:
later we can change it to the hex version

Reply:
Aah, I can do it now